bigtable/bigtable.go

```diff
@@ -923,6 +925,40 @@ func handleExecuteStreamEnd(stream btpb.Bigtable_ExecuteQueryClient, trailerMD *
 	return nil
 }
 
+// Pinger pings the server and warms up the connection.
+func (c *Client) Pinger(ctx context.Context) (err error) {
+	md := metadata.Join(metadata.Pairs(
+		resourcePrefixHeader, c.fullInstanceName(),
+		requestParamsHeader, c.reqParamsHeaderValInstance(),
```

Comment:
what's the value of `c.reqParamsHeaderValInstance()`? PingAndWarm has slightly different request params from other methods. It's in the format of `name=<instance_name>&app_profile_id=<app_profile_id>`